Galaxus.Functional.Tests/(Result)/TExtensionsForResultTest.cs

```diff
@@ -158,5 +162,437 @@ public async Task OrElseAsync_SynchronousContinuation_ReturnsSuccessTask()
             await resultTask;
             Assert.AreEqual(expected: TaskStatus.RanToCompletion, actual: resultTask.Status);
         }
+
+
+        [Test]
```

Comment:
Now I'm curious - the UnwrapAsync for Option was deserving of its own test file, but UnwrapAsync for Result is not? :P

Reply:
Ok I splitted now all Tests von TExtensionsForResultTest.cs in separate files. But honestly, there is kind of a mess in this library and we should really invest some time in a clear structure and some style guides.